doc/cdi-config.md

```diff
@@ -25,4 +28,6 @@ Currently it is used only for holding Upload Proxy URL details.
 | filesystemOverhead      |                       | updated when the spec values are updated, to show the per-storageClass calculated result as well as the per-storageClass one. |
 |   global                | "0.055"               | The calculated overhead to be used for all storageClasses unless a specific value is chosen for this storageClass |
 |   storageClass          |                       | The calculated overhead to be used for every storageClass in the system, taking into account both global and per-storageClass values. |
-
+| preallocation           |                       | updated when the spec values are updated, to show the per-storageClass calculated result as well as the per-storageClass one. |
```

Comment:
I do not understand this part "to show the per-storageClass calculated result as well as the per-storageClass one."